example.html

```diff
@@ -120,10 +120,9 @@
       </div>
     </div>
 
-    <script src="butr-to.min.js"></script>
-    <script src="butr-marker.min.js"></script>
+    <script src="dist/butr.js"></script>
     <script>
-      Butr.marker({ useTo: true })
+      Butr.marker({ useTo: true });
```

Comment:
lollllllllll @friartuck6000

Reply:
Hahahaha, didn't even realize I did that